### ISSUE TITLE
[Feature] Spectrum materials

### ISSUE THREAD

**Issue by Mieridduryn** [KonSola5/HephaestusPlus#12]
## **Material compatibility request**
Spectrum is a magic mod by Dafuqsy, and it adds several new materials. Here are some that I think should be able to be made into tool parts:
- Azurite
- Malachite
- Bloodstone
- Dragonbone or bone ash
- Pyrite
- Bismuth
- Blackslag
- Paltaeria
- Stratine
- Noxwood
- Downstone
- Neolith

## **This section contains lore spoilers**
When Amethyst, Citrine, and/or Topaz are melted down, they could melt into three fluids called "Dreiton Blood."

**Comment by KonSola5** [comment]
Thanks for the suggestion! Spectrum was on the roadmap, though the suggestion will make it easier to set up good traits for these materials.

Maybe for gems, they will be alloyed with metals to create an alloy, just like Amethyst Bronze.

**Comment by Mieridduryn** [comment]
Sounds like a good idea! There's themings to each gem, such as amethyst/magenta being Time, topaz/cyan being Matter, and citrine/yellow being Energy. Perhaps you could go off of that, a little.

**Comment by Mieridduryn** [comment]
Dragonbone/bone ash could be used for a new type of reinforcement, and perhaps downstone could be used to make a tool indestructible (as it is used to craft the Indestructible enchant with Spectrum).

**Comment by Mieridduryn** [comment]
You also have the opportunity to use molten metals/gems for some custom Fusion Shrine recipes, or allow alloying in the Fusion Shrine similar to how Netherite can be made from pure debris & gold. There aren't any pure resources for stuff like tin, zinc, lead, etc. in Spectrum, but I've proposed it to the devs and they seem on board with the idea (though it's not guaranteed to be implemented). At the very least, perhaps pure iron + pure coal could be made into steel or something like that.

**Comment by KonSola5** [comment]
I don't think pure alloys fit in HephaestusPlus.

Though HephaestusPlus can add the ability to melt pure resources and turn them into some amount of normal resource.

**Comment by KonSola5** [comment]
Pure Cobalt can be added though.

**Comment by Mieridduryn** [comment]
True. How about adding resonance shards in the Tinker's anvil for a resonance-imitating modifier? Hehe

**Comment by KonSola5** [comment]
Modifiers mimicking Spectrum enchants are planned.
